### PR TITLE
Add Ubuntu 24.04 LTS

### PR DESCRIPTION
**Pull request by mayrstefan**
# Description

Add Ubuntu 24.04 LTS when it is released an the PR for Uyuni is merged

# Target branches

* Which product version this PR applies to: Uyuni
* Needs to be reviewed for SUMA and added

Backport targets (edit as needed):
- master
- 4.3

# Links
- Related development PR [#8504](https://github.com/uyuni-project/uyuni/pull/8504)
- Related Documentation issue: https://github.com/SUSE/spacewalk/issues/23922

### CODE REVIEW

**Comment by 0rnela** [comment]
Thank you @mayrstefan. We are creating the formal issues for adding the client to the documentation.

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+* {ubuntu} 20.04, 22.04 and 24.04 repository URLs are available from {scc}
 * Packages and metadata are provided by Ubuntu, not by {suse}
 * For supported products, see the support table and the release notes
 ====
 endif::[]
 
 ifeval::[{suma-content} == true]
-{productname} supports {ubuntu} 20.04 LTS and 22.04 LTS clients using {salt}.
+{productname} supports {ubuntu} 20.04 LTS, 22.04 LTS and 24.04 LTS clients using {salt}.
 //For information about registering Salt clients running {ubuntu} 18.04 LTS, see xref:client-configuration:clients-ubuntu-old.adoc[].
```
Comment (raulillo82) [inline modules/client-configuration/pages/clients-ubuntu.adoc:18]:
```suggestion
```
This comment line seems not to exist any more in master.

File: modules/client-configuration/pages/supported-features-ubuntu.adoc [hunk 2]
```diff
@@ -18,183 +18,224 @@ The icons in this table indicate:
...
 | {cross}
 | {cross}
+| {cross}
 
 | Recurring Actions
 | {check}
 | {check}
-| {salt}
-| {salt}
+| {check}
```
Comment (raulillo82) [inline modules/client-configuration/pages/supported-features-ubuntu.adoc:228]: There's a conflict here right now, which should disappear when you rebase on top of the latest master, if I'm not mistaken. These `| {salt}` lines are not there any more right now.

**Review by raulillo82 — APPROVED** [review]
Other than my two comments regarding merging conflicts, it looks good to me. However I will need an extra review from Docs team, I'll reach them today meanwhile.
In the meantime, @mayrstefan , can you maybe rebase this again? Probably the merging conflicts will disappear them (or you can easily solve them). Thanks a lot in advance for the contribution, I'm pushing to include this (and the main PR together with this).

**Comment by keichwa** [comment]
> Other than my two comments regarding merging conflicts, it looks good to me. However I will need an extra review from Docs team, I'll reach them today meanwhile. In the meantime, @mayrstefan , can you maybe rebase this again? Probably the merging conflicts will disappear them (or you can easily solve them). Thanks a lot in advance for the contribution, I'm pushing to include this (and the main PR together with this).

Yes, to me it also looks good.  It looks as if you could also the github UI (webeditor) to solve the conflicts.

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]
Please backport to **manager-5.0** (Or we can).

**Review by 0rnela — APPROVED** [review]

**Review by keichwa — APPROVED** [review]

**Pull request merged**